README.md

```diff
@@ -58,9 +65,42 @@ Then, [build an APK](https://developer.android.com/studio/build/building-cmdline
 deltachat@6012dcb974fe:/home/app$ ./gradlew assembleDebug
 ```
 
-If you don't want to use Docker or Podman, proceed to the next section.
+## Troubleshooting
+
+- `./gradlew assembleDebug` fails with `The SDK directory '/home/user/Android/Sdk' does not exist.`:
+  
+  You have to
+  - either: remove the line starting with `sdk.dir=` from your `local.properties` file
+  - or: execute `./gradlew assembleDebug` from outside the container
+
+# <a name="setup-podman"></a>Setup Podman
+
+These instructions were only tested on a Manjaro machine so far. If anything doesn't work, please open an issue.
+
+First, [Install Podman](https://podman.io/getting-started/installation).
+
+Then in /etc/containers/storage.conf, replace the line
+```
+driver = ""
+```
+with:
+```
+driver = "overlay"
+```
+You can also set the `driver` option to something else, you just need to set it to _something_.
```

Comment:
I don't have the file `/etc/containers/storage.conf` at all on my Debian Sid. Can't check my Arch setup right now, but I don't remember setting this either.

Reply:
I'll move it to the "Troubleshooting" section then